imports/plugins/included/product-detail-simple/lib/helpers/getDisplayPriceByCurrency.js

```diff
@@ -0,0 +1,12 @@
+/**
+ *
+ * @method getDisplayPriceByCurrency
+ * @summary Gets the display price per given pricing array from a product, variant, or option
+ * @param {Array} pricingArr - An array of pricing info
+ * @param {Array} currencyCode - Selected currency of customer
+ * @return {String} - Display price of a product, variant, or option
+ */
+export default function getDisplayPriceByCurrency(pricingArr, currencyCode = "USD") {
+  const currencyPricing = pricingArr.filter((pricing) => pricing.currency.code === currencyCode);
```

Comment:
@ajporlante This could be `pricingArray.find` so that you don't have to do `currencyPricing[0]`.